CMakeLists.txt

```diff
@@ -3,16 +3,26 @@ cmake_minimum_required(VERSION 3.0)
 project(ssb-dbgen # SSB benchmark data (and query) generator utilities
 	LANGUAGES C)
 
+set(CMAKE_C_FLAGS_DEBUG "-g")
```

Comment:
CMake adds this by default for the Debug build type. However, we _do_ need to make sure that a build type is indeed set.

Reply:
... and now I've done so.